src-tauri/src/bidder/mod.rs

```diff
@@ -54,21 +56,71 @@ impl Bidder {
         Ok(())
     }
 
+    pub async fn update_bot_if_needed(app: &AppHandle, ssh: &SSH) -> Result<()> {
+        if Self::needs_new_bot_version(ssh, app).await? {
+            ssh.run_command("mkdir -p commander-config/bot")
+                .await?;
+            ssh.upload_directory(app, "bot".into()).await?;
+            ssh.upload_directory(app, "bot/src".into()).await?;
+            Self::start_docker(ssh).await?;
+        }
+        Ok(())
+    }
+
+    async fn needs_new_bot_version(ssh: &SSH, app: &AppHandle) -> Result<bool> {
+        let remote_version = match ssh
```

Comment:
this is better named as remote_package_json. No need to update. Just marking it.